xinference/model/image/stable_diffusion/core.py

```diff
@@ -45,20 +45,19 @@ def __init__(
         self._model_path = model_path
         self._device = device
         self._model = None
-        self._lora_model_path = lora_model_path
+        self._lora_model_paths = lora_model_paths
         self._lora_load_kwargs = lora_load_kwargs or {}
         self._lora_fuse_kwargs = lora_fuse_kwargs or {}
         self._kwargs = kwargs
 
     def _apply_lora(self):
-        if self._lora_model_path is not None:
+        if self._lora_model_paths is not None:
             logger.info(
                 f"Loading the LoRA with load kwargs: {self._lora_load_kwargs}, fuse kwargs: {self._lora_fuse_kwargs}."
             )
             assert self._model is not None
-            self._model.load_lora_weights(
-                self._lora_model_path, **self._lora_load_kwargs
-            )
+            for path in self._lora_model_paths:
+                self._model.load_lora_weights(path, **self._lora_load_kwargs)
```

Comment:
Did you try this? Is this correct? First load all lora weights, then just fuse_lora?

Reply:
The model qwen14B, using both [peft_qwen_14b_chat_int4_xlsum](https://huggingface.co/p1atdev/peft_qwen_14b_chat_int4_xlsum) and [Qwen-14b-chat-32k-lora](https://huggingface.co/yuyijiong/Qwen-14b-chat-32k-lora), yields the following results:
![image](https://github.com/xorbitsai/inference/assets/112739514/0e31ba88-0cf0-4cc7-80c8-a1d149bd8d61)